src/restore.rs

```diff
@@ -104,7 +112,16 @@ pub fn restore(
     )?;
     for entry in entry_iter {
         if options.print_filenames {
-            progress_bar.message(&format!("{}\n", entry.apath()));
```

Comment:
Ideally I should unify the things that print like this, but this PR is already big, so not here.